elastic/kibana
# decouple url drilldown action from Embeddable framework

Part of https://github.com/elastic/kibana/issues/175138 and prerequisite for https://github.com/elastic/kibana/issues/174960

PR decouples Url drilldown action from Embeddable framework by migrating to sets of composable interfaces.

### test instructions
1. Create panel and add "Url drilldown"
2. Verify `context` variables are populated with the same values when they were grabbed from embeddable.input and embeddable.output
    <img width="600" alt="Screenshot 2024-01-31 at 2 11 20 PM" src="https://github.com/elastic/kibana/assets/373691/150f7a61-911f-4fb6-bf85-5c0481865e4b">
